binderhub/traitlets.py

```diff
@@ -0,0 +1,18 @@
+from traitlets import TraitType
```

Comment:
Is it a good idea to create an internal package with the same name as an external package we import?

I'm worried that one day we will spend forever trying to see the difference between `.traitlets` and `traitlets` :-/

However I don't have a good alternative name as a suggestion.

Reply:
To be honest I didn't think about it, I just copied it from jupyterhub repo (https://github.com/jupyterhub/jupyterhub/blob/master/jupyterhub/traitlets.py).

But when you say it, I understand. Should we keep it same with jupyterhub or I can rename it to `extra_traitlets.py`?

Reply:
If JupyterHub does it then maybe it isn't as much of a trap as I expect it to be. Let's keep it this way. Then at least it is a consistent trap :D